src/components/call/CallPageContainer.tsx

```diff
@@ -68,7 +71,8 @@ const mapStateToProps = (state: ApplicationState, ownProps: OwnProps): StateProp
   return {
     callState: state.callState,
     currentIssue: currentIssue,
-    splitDistrict: state.locationState.splitDistrict
+    splitDistrict: state.locationState.splitDistrict, // todo: just pass locationState and remove separate splitDistrict
```

Comment:
Good `todo` here, definitely doesn't need to be refactored with the other changes.